usmqe/api/tendrlapi/tendrlapi.py

```diff
@@ -0,0 +1,214 @@
+        """
+        pattern = "{}/GlusterCreateVolume".format(cluster)
+        response = requests.post(pytest.config.getini("usm_api_url") + pattern,
+                                 json=volume_data)
+        asserts = {
+            "reason": 'Accepted',
+            "status": 202,
+        }
+        self.print_req_info(response)
+        self.check_response(response, asserts)
+        return response.json()
+
+    def delete_volume(self, cluster, post_data):
+        """ Import gluster cluster defined by json.
+
+        Name:        "delete_volume",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/GlusterDeleteVolume",
+
+        Args:
+            cluster: id of a cluster where will be created volume
+            volume_data: json structure containing data that will be sent to api server
+        """
+        pattern = "{}/GlusterDeleteVolume".format(cluster)
+        response = requests.post(pytest.config.getini("usm_api_url") + pattern,
+                                 json=post_data)
+        asserts = {
+            "reason": 'Accepted',
+            "status": 202,
+        }
+        self.print_req_info(response)
+        self.check_response(response, asserts)
+        return response.json()
+
+    def start_volume(self, cluster, volume_data):
+        """ Start gluster volume specified by json.
+
+        Name:        "start_volume",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/GlusterStartVolume",
+
+        Args:
+            cluster: id of a cluster where will be created volume
+            volume_data: json structure containing data that will be sent to api server
+        """
+        pattern = "{}/GlusterStartVolume".format(cluster)
+        response = requests.post(pytest.config.getini("usm_api_url") + pattern,
+                                 json=volume_data)
+        asserts = {
+            "reason": 'Accepted',
+            "status": 202,
+        }
+        self.print_req_info(response)
+        self.check_response(response, asserts)
+        return response.json()
+
+    def stop_volume(self, cluster, volume_data):
+        """ Stop gluster volume specified by json.
+
+        Name:        "stop_volume",
+        Method:      "POST",
+        Pattern:     ":cluster_id:/GlusterStartVolume",
+
+        Args:
+            cluster: id of a cluster where will be created volume
+            volume_data: json structure containing data that will be sent to api server
+        """
+        pattern = "{}/GlusterStopVolume".format(cluster)
+        response = requests.post(pytest.config.getini("usm_api_url") + pattern,
+                                 json=volume_data)
+        asserts = {
+            "reason": 'Accepted',
+            "status": 202,
+        }
+        self.print_req_info(response)
+        self.check_response(response, asserts)
+        return response.json()
+
+    def check_volume_attribute(self, cluster, volume, attribute, value, positive=True):
```

Comment:
I think that will be more readable if this wrapper will be unwrapped directly in test.